### ISSUE TITLE
Change instances of `Clavis Sequentiarium` to `Clavis Sequentiarum`

### ISSUE THREAD

**Issue by dchiller** [DDMAL/CantusDB#1544]
In the dropdowns under Chants and Sources, we have "Sequentiarium". In the header on https://cantusdatabase.org/sequences/, we have "Sequentiarum". 

This discrepancy was pointed out to me a while back but I failed to note which one was the issue.

@annamorphism ?

**Comment by annamorphism** [comment]
"Sequentiarum" (no I at the end) is correct. (The other option is also a valid word, but not grammatical in this context...)

**Comment by dchiller** [comment]
Thanks @annamorphism
